### PR TITLE
Fission/update sector agricultural extension

### PR DESCRIPTION
**Pull request by hemanthReddy06**

### CODE REVIEW

File: dimagi/pages/static/style/pages/sectors.scss [hunk 1]
```diff
@@ -62,3 +62,46 @@ a.study {
     }
   }
 }
+
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors.scss]: please delete commented code that is no longer being used.

File: dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load dimagidotcom %}
+
+{% block title %}{% blocktrans %}
+	Agricultural Extension Programs
```
Comment (biyeun) [inline dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html]: 2 space indentation

File: dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  {% compress css %}
+    <link href="{% sass_src 'style/pages/sectors/agricultural.scss' %}" rel="stylesheet" media="screen" type="text/css" />
+  {% endcompress %}
+{% endblock %}
+
+{% block hero %}  
+{% endblock %}
+
+{% block content %}
+	{% include "sections/sectors/agriculture_extension_programs/sub_nav.html" %}
```
Comment (biyeun) [inline dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html]: 2 space indentation

File: dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	{% include "sections/sectors/agriculture_extension_programs/feature.html" %}
+	{% include "sections/sectors/agriculture_extension_programs/part3.html" %}
+	{% include "sections/sectors/agriculture_extension_programs/news.html" %}
+	{% include "sections/sectors/agriculture_extension_programs/part4.html" %}
+	{% include "sections/sectors/agriculture_extension_programs/testimonial.html" %}
+{% endblock %}
+
+{% block modals %}
+	<div class="modal" 
+			 data-dismissible="true"
```
Comment (biyeun) [inline dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html]: fix indentation

File: dimagi/pages/templates/sections/sectors/agriculture_extension_programs/benefit.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
+{% load i18n %}
+{% load dimagidotcom %}
+<section class="" 
+         data-lazybg="true">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/sectors/agriculture_extension_programs/benefit.html]: fix indentation. 2 space indentation for HTML files.

File: dimagi/pages/templates/svg/sectors/agriculture-extension-programs/AgExtension_Icon1.html [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    <g id="1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round">
+        <g id="Icon-3" transform="translate(7.000000, 7.000000)" stroke="#47b700" stroke-width="2">
+            <path d="M18.7,41.7 C20.8719244,39.5205441 23.8231064,38.2968832 26.9,38.3 C29.9806651,38.2777347 32.9389605,39.504345 35.1,41.7" id="Path"></path>
+            <path d="M12.8,35.7 C16.5704157,31.9382012 21.673946,29.8177203 27,29.8 C32.3353408,29.771037 37.4561139,31.8986821 41.2,35.7" id="Path"></path>
+            <path d="M6.2,29.1 C11.6810554,23.5325281 19.1875964,20.4288621 27,20.5 C34.7993917,20.495231 42.2812877,23.5887072 47.8,29.1" id="Path"></path>
+            <path d="M0,22.9 C7.16872347,15.7611119 16.8830212,11.7674562 27,11.8 C37.1077568,11.8142531 46.8049175,15.8008636 54,22.9" id="Path"></path>
+            <path d="M8.6,0 L35.6,54.2" id="Path"></path>
+        </g>
+    </g>
+</svg>
```
Comment (biyeun) [inline dimagi/pages/templates/svg/sectors/agriculture-extension-programs/AgExtension_Icon1.html]: empty new line at end of svg files please. also make sure that the 2 space indent rule is also obeyed in these files

File: dimagi/pages/urls/sectors.py [hunk 7]
```diff
@@ -3,12 +3,15 @@
 from dimagi.pages.views.sectors import (
     view_all,
     view_single,
+    agriculture_extension_programs
```
Comment (biyeun) [inline dimagi/pages/urls/sectors.py]: add comma at the end of this line (pylint)

File: dimagi/pages/static/style/pages/sectors.scss [hunk 8]
```diff
@@ -61,4 +61,4 @@ a.study {
       opacity: .5;
     }
   }
-}
+}
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors.scss]: new line needed at end of file

File: dimagi/pages/static/style/pages/sectors/agricultural.scss [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+  @include make-bg(hero-data-image, '../../../pages/sectors/agriculture-extension-programs/intro/hero', $_base-height, $_height-by-size);
+}
+
+.news-images .image.blog1 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/news/AgExtension_Blog1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.news-images .image.blog2 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/agricultural.scss]: new line between style definitions please

File: dimagi/pages/static/style/pages/sectors/agricultural.scss [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+  background-size: contain;
+  background-position: left;
+}
+.news-images .image.blog3 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/news/AgExtension_Blog3.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.programs-images .image.featured-project1 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/agricultural.scss]: new line between style definitions please

File: dimagi/pages/static/style/pages/sectors/agricultural.scss [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+  background-size: contain;
+  background-position: left;
+}
+.news-images .image.blog2 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/news/AgExtension_Blog2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.news-images .image.blog3 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/agricultural.scss]: new line between style definitions please

File: dimagi/pages/static/style/pages/sectors/agricultural.scss [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+  background-size: contain;
+  background-position: left;
+}
+.programs-images .image.featured-project1 {
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/programs/AgExtension_FeaturedProject1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.programs-images .image.featured-project2 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/agricultural.scss]: new line between style definitions please

File: dimagi/pages/static/style/pages/sectors/agricultural.scss [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+  @include background-cover('../../../pages/sectors/agriculture-extension-programs/programs/AgExtension_FeaturedProject2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.programs-images .image.featured-project3 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/agricultural.scss]: new line between style definitions please

File: dimagi/pages/static/style/pages/sectors/agricultural.scss [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+  @media (min-width: $width-desktop) {
+    .off-grid {
+      top: 10%;
+      left: 0;
+      position: absolute;
+      right: auto;
+      width: 120%;
+      z-index: 1;
+    }
+    .content {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/agricultural.scss]: new line between style definitions please

File: dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+{% endblock %}
+
+{% block js %}
+  <!--[if lte IE 8]>
+    <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
+  <![endif]-->
+  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+  <script>
+    //  Schedule demo form
+    hbspt.forms.create({
```
Comment (biyeun) [inline dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html]: is there a reason why the demo form is being activated here and not through `demoForm.js`? seems like a lot of duplicated code in all of these PRs

File: dimagi/pages/templates/sections/sectors/agriculture_extension_programs/news.html [hunk 16]
```diff
@@ -0,0 +1,170 @@
...
+              {% endblocktrans %}
+            </a>
+          </p>
+          </div> {# .pad-line #}
+        </div> {# .white-bg #}
+      </div> {# .column #}
+    </div>{# .grid #}
+  </div>{# .content #}
+  <div class="content pad-top-small">
+    <h2 class="">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/sectors/agriculture_extension_programs/news.html]: remove empty class attribute

File: dimagi/pages/templates/sections/sectors/agriculture_extension_programs/programs.html [hunk 17]
```diff
@@ -0,0 +1,235 @@
...
+        {% blocktrans %}
+          Featured CommCare-Based Agricultural <br class="hide-until-medium"> Extension Programs
+        {% endblocktrans %}
+      </h2>
+      <div class="grid grid-3-at-large grid-1-at-medium grid-row-equal-heights grid-col-space-small">
+        <div class="column">
+          <div class="pad-bottom-base white-bg">
+            <div class="image featured-project1" data-lazybg="true"></div>
+            <div class="pad-line">
+              <h5 class=""><strong class="body-color">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/sectors/agriculture_extension_programs/programs.html]: remove empty class attribute. move strong to new line

File: dimagi/pages/templates/sections/sectors/agriculture_extension_programs/programs.html [hunk 18]
```diff
@@ -0,0 +1,235 @@
...
+                </a>
+              </p>
+            </div> {# .pad-line #}
+          </div> {# .white-bg #}
+        </div> {# .column #}
+        <div class="column">
+          <div class="pad-bottom-base white-bg">
+            <div class="image featured-project2" data-lazybg="true"></div>
+            <div class="pad-line">
+              <h5 class="">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/sectors/agriculture_extension_programs/programs.html]: remove empty class attribute

File: dimagi/pages/templates/sections/sectors/agriculture_extension_programs/programs.html [hunk 19]
```diff
@@ -0,0 +1,235 @@
...
+                </a>
+              </p>
+            </div> {# .pad-line #}
+          </div> {# .white-bg #}
+        </div> {# .column #}
+        <div class="column">
+          <div class="pad-bottom-base white-bg">
+            <div class="image featured-project3" data-lazybg="true"></div>
+            <div class="pad-line">
+              <h5 class="">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/sectors/agriculture_extension_programs/programs.html]: remove empty class attribute

File: dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html (hunk 15 above)
Comment (sayyedhanif1) [reply dimagi/pages/templates/pages/sectors/agriculture_extension_programs.html]: as I got diff formId. The given form id for this page is not matched as given in demoForm js file . So still we want to use demoForm.js or need to create new js for the same?

File: dimagi/pages/static/style/pages/sectors/agricultural.scss [hunk 20]
```diff
@@ -0,0 +1,123 @@
...
+.part3 .image .bg-loader{
+  background-color: transparent;
+}
+
+.mb-10 {
+  margin-bottom: 10px;
+}
+
+.touch .sub-nav a:not(.button).primary span, .no-touch.sub-nav-fixed-top .sub-nav a.primary span {
+  background-color:#47b700;
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/agricultural.scss]: hex colors should never be referenced directly. please, please, please use the variables in `colors.scss`.

**Comment by biyeun** [comment]
also, for the phone image issue, run 
```
./manage.py build_thumbnails
```
locally, and then you will see the "ghost" image. I believe you are running into this issue because you are not using the phone image like we have been using it in the [existing case management blurb](https://github.com/dimagi/dimagi.com/blob/5571e46c05ee0ba8bc6454f3a9ca21ef207b7259/dimagi/pages/templates/sections/case_management/greater_impact.html#L82)

**Pull request merged**